packages/@aws-cdk/cdk/lib/program.ts

```diff
@@ -0,0 +1,176 @@
+import cxapi = require('@aws-cdk/cx-api');
+import fs = require('fs');
+import path = require('path');
+import { IConstruct, Root } from './core/construct';
+import { merge } from './util/json';
+import { validateAndThrow } from './util/validation';
+
+/**
+ * Represents a CDK program.
```

Comment:
Program and app feel too interchangeable. Would rather call this `Host` or something more distinctive. No hard feelings though.